plugins/filter_kubernetes/kube_meta.c

```diff
@@ -87,6 +87,63 @@ static int file_to_buffer(const char *path,
     return 0;
 }
 
+/* Run command to get token when token file cannot open */
+static int get_token_with_command(const char *command, 
+                                  char **out_buf, size_t *out_size)
+{
+    FILE *fp;
+    char buf[4096];
```

Comment:
How did we decide the size? Is is guaranteed to be less than the allocated buffer. Also, we used this number in two places. Can we write it in a CONSTANT variable?

Reply:
We just choose a large number because there is no maximum length for a token. Now we will get a token over 2000 bytes if we run the command. So we just double it. But my code also helps to append the token if its length is longer than 4096. You can see my code below.